legal-api/report-templates/template-parts/bc-address-change/addressChangeDetails.html

```diff
@@ -17,6 +17,10 @@
               <td>{{effective_date_time}}</td>
             </tr>
           </table>
+          <div class="filing-identifier">
```

Comment:
Should this block be added to new directorChangeDetails.html also?

Reply:
Yes